app/graphql/types/query_type.rb

```diff
@@ -15,5 +23,14 @@ def get_users
     def get_user(id:)
       User.find(id)
     end
+
+    # Event queries
+    def get_events
+      Event.all
+    end
+
+    def get_event(id:)
+      Event.find(id)
+    end
```

Comment:
Great job on this.  I'm still trying to wrap my head around all this, and this helps!